docs/cli-context/README.adoc

```diff
@@ -0,0 +1,240 @@
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-setting-services-cli_getting-started-rhoas-context[Set services in service context]
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-generating-config-cli_getting-started-rhoas-context[Generating application configurations for Quarkus]
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-switching-context-cli_getting-started-rhoas-context[Switch between contexts]
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-unsetting-services-cli_getting-started-rhoas-context[Unset services in service context]
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-deleting-context-cli_getting-started-rhoas-context[Delete a service context]
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-sharing-context-cli_getting-started-rhoas-context[Sharing contexts]
+
+
+[id="ref-kafka-cli-prereqs_{context}"]
+== Prerequisites
+
+[role="_abstract"]
+* You've installed the `rhoas` CLI. For more information, see {base-url}{rhoas-cli-installation-url}[_Installing the rhoas CLI_^].
+
+[id="proc-creating-context-cli_{context}"]
+== Creating a service context
+
+.Procedure
+
+Enter the following command to create a context named `my-context`
+
+.Creating a context
+[source,shell]
+----
+$ rhoas context create --name my-context
+----
+
+[NOTE]
+====
+By default, the created context is set as the current context.
+All the service based commands are executed against the service instances set in the current context.
+====
+
+[id="proc-setting-services-cli_{context}"]
+== Setting services in a context
+
+[role="_abstract"]
+Context contains identifiers to service instances.
+All the service based commands are executed against the service instances set in the current context.
+
+.Procedure
+
+. Setting Kafka Instance in the current context
++
+--
+.Set Kafka instance in the current context
+[source,shell]
+----
+$ rhoas context set-kafka
+----
+--
+
+. Setting Service Registry Instance in the current context
++
+--
+.Set Service Registry instance in the current context
+[source,shell]
+----
+$ rhoas context set-service-registry
+----
+
+[NOTE]
+====
+Creating a new service instance automatically sets it in the current context.
+====
+--
+
+. Check status of the current context
++
+--
+After setting service(s) in the current context, their staus can be checked using the `rhoas context status` command
+.View status of the current context
+[source,shell]
+----
+$ rhoas context status
+----
+--
+
+[id="proc-generating-config-cli_{context}"]
+== Generating application configurations for contexts
+
+`rhoas generate-config` command generates application configurations that can be used to connect to the services set in context.
+
+.Generating configurations for context named as my "my-context"
+source,shell
+----
+$ rhoas generate-config --name my-context  -type json
+----
+
+=== Generating configurations for quarkus applications
```

Comment:
@wtrocki I think we can add something similar for Nodejs here as well at a later stage as well?